src/Illuminate/Foundation/Testing/TestResponse.php

```diff
@@ -887,9 +887,19 @@ public function assertSessionHasErrors($keys = [], $format = null, $errorBag = '
      *
      * @return $this
```

Comment:
Docblocks?

Reply:
Good catch, forgot to update them, thanks